### PR TITLE
feat: update kernel with sfc driver and LSM updates

### PR DESCRIPTION
**Pull request by smira**
See:

* https://github.com/siderolabs/pkgs/pull/890
* https://github.com/siderolabs/pkgs/pull/891

### CODE REVIEW

**Review by utkuozdemir — APPROVED** [review]

**Review by frezbo — APPROVED** [review]
also on a side note, should this be an extension, i meant all the sfc modules?

**Comment by smira** [comment]
> also on a side note, should this be an extension, i meant all the sfc modules?

it feels like so far we put network/storage drivers to the core, but I'm open to the options here

**Review by Unix4ever — APPROVED** [review]

File: hack/modules-amd64.txt [hunk 1]
```diff
@@ -1,6 +1,6 @@
-modules.order
 modules.builtin
 modules.builtin.modinfo
+modules.order
```
[the remark this answers is not in the document]
Comment (smira) [reply hack/modules-amd64.txt]: sorted them all

**Pull request merged**